src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/BlockChangeMulti.java

```diff
@@ -18,6 +22,14 @@ public BlockChangeMulti(ConnectionImpl connection) {
 
 	protected final TileEntityRemapper tileremapper = TileEntityRemapper.getRemapper(connection.getVersion());
 
+	private static ClientBoundPacketData createPacket(ProtocolVersion version, TileEntity tile) {
+		ClientBoundPacketData serializer = ClientBoundPacketData.create(ClientBoundPacket.PLAY_UPDATE_TILE_ID);
```

Comment:
Huh? This seems like the wrong place for this packet? The problem is that the import used to be wrong I think.
The solution is to fix the import, not make a copy of that method here I think.